async-nats/src/jetstream/consumer/pull.rs

```diff
@@ -205,26 +196,16 @@ impl Consumer<Config> {
     ///     ..Default::default()
     /// }).await?;
     ///
-    /// let mut messages = consumer.batch(100, None).await?;
+    /// let mut messages = consumer.batch().max_messages(100).messages().await?;
     /// while let Some(Ok(message)) = messages.next().await {
     ///   println!("got message {:?}", message);
     ///   message.ack().await?;
     /// }
     /// Ok(())
     /// # }
     /// ```
-    pub async fn batch(&self, batch: usize, expires: Option<usize>) -> Result<Batch, Error> {
-        Batch::batch(
-            BatchConfig {
-                batch,
-                expires,
-                no_wait: false,
-                idle_heartbeat: Duration::default(),
-                ..Default::default()
-            },
-            self,
-        )
-        .await
+    pub fn batch(&self) -> BatchBuilder {
```

Comment:
So the distinction now, between fetch and batch is no_wait default which is not configurable?

Reply:
yes.